UIAutomationTests/TestsGeneralFunctionality.cs

```diff
@@ -32,7 +32,7 @@ public void OdbcDatasourceOverview()
         }
         //metadata test - adventure works (populated schema)
         [Fact]
-        public void OdbcDatasourcePopulatedSchemaAdventureWorksRows()
+        public void OdbcDatasourcePopulatedSchemaRows()
```

Comment:
```suggestion
        public void TestOdbcDatasourcePopulatedSchemaRows()
```

and same with other tests